app/controllers/AnnotationIOController.scala

```diff
@@ -411,7 +411,7 @@ Expects:
           skeletonAnnotationLayer =>
             tracingStoreClient.getSkeletonTracing(skeletonAnnotationLayer, skeletonVersion)
         } ?~> "annotation.download.fetchSkeletonLayer.failed"
-        user <- userService.findOneCached(annotation._user) ?~> "annotation.download.findUser.failed"
+        user <- userService.findOneCached(annotation._user)(GlobalAccessContext) ?~> "annotation.download.findUser.failed"
```

Comment:
So, this failed before (ignoring the caching issue) because the requesting user didn't have access to the owning user?
Does this mean that an annotation owned by user A can be downloaded by user B (if B has access to it) even though user B might not be allowed to "read" user A? Because user A needs to be serialized into the annotation?
I'm wondering whether this circumvents the permission system a bit?

Reply:
Yes, that’s true. However, not all info about user A is serialized into the annotation. I believe we had a discussion about this some time ago and decided that it would be fair to say that a user publishing an annotation publishes their name with it. I’m open to having that discussion again, though. Also it might be worth double-checking what info is really needed for wk to show an annotation. The download only includes the name, but i think the annotation info request (which already works in the same way before this PR) also includes the email address.

Reply:
Ok, fair enough :+1:

> Also it might be worth double-checking what info is really needed for wk to show an annotation.

Yes, this is a good idea. One could also put in a dummy email value à la `anonymous`.

Reply:
https://github.com/scalableminds/webknossos/issues/7029